index.html

```diff
@@ -265,7 +285,12 @@ <h1 class="myself-header">
             </ul>
           </nav>
```

Comment:
- [ ] On line 253, use a header tag of `h2` instead of `h1` to ensure a meaningful hierarchy for the headers is implemented.